static/app/views/performance/data.tsx

```diff
@@ -450,7 +450,9 @@ function generateGenericPerformanceEventView(
   savedQuery.query = conditions.formatString();
 
   const eventView = EventView.fromNewQueryWithLocation(savedQuery, location);
-  eventView.additionalConditions.addFilterValues('event.type', ['transaction']);
+  if (!isMetricsData) {
```

Comment:
This seems to be doing some product/business related logic, so I would probably add a comment in all the places where we do something like this and on explaining why (unless it's super obvious and I just dont have enough knowledge yet 😅)

Reply:
yes sure... I'll add a comment 😉